src/org/opendatakit/briefcase/ui/reused/source/Source.java

```diff
@@ -408,6 +482,16 @@ public String getDescription() {
       return String.format("%s at %s", form.getFormName(), path.toString());
     }
 
+    @Override
+    public List<String> pullScriptLines(List<FormStatus> forms, BriefcasePreferences appPreferences) {
+      throw new BriefcaseException("Not yet implemented for FormInComputer");
+    }
+
+    @Override
+    public List<String> pushScriptLines(List<FormStatus> forms, BriefcasePreferences appPreferences) {
+      throw new BriefcaseException("Not yet implemented for FormInComputer");
```

Comment:
Once we change what this method returns, maybe we won't need to throw an exception. Maybe we can return the internal state of this instance and let the calling site decide what to do with it.